### ISSUE TITLE
Searching for Projects v11 #77

### ISSUE THREAD

**Issue by ChrisJarka** [unfoldingWord-dev/ts-desktop#925]
tS-D v11 build 77

In the Search for Projects window (Import Options - Import from Door43) it would be nice to have the date of upload displayed. (Same date displayed in Gogs.)

And - ENHANCEMENT - a sort option to sort by username or date or project name.
![search order](https://cloud.githubusercontent.com/assets/9850185/20848340/b3f4bdd6-b89f-11e6-9999-eff335e976a5.png)

**Comment by sea-liesch** [comment]
I agree having another search criteria ( like date) would be helpful.   One of the comments I've gotten is "which project was last worked on?"  Being able to sub sort by date would be a welcomed addition.

**Comment by cdwhitfield73** [comment]
So it appears that it is not currently sorting by date or anything for that matter.  And the date info is not even being passed to the app.  So adding any additional fields to display, or search by, and adding any kind of sorting to this list would be a significant feature addition that will have to be discussed for versions after 11.

**Comment by ChrisJarka** [comment]
@cdwhitfield73 yes, please, add this as an enhancement for future versions. It sounds like the field would want the latest to show at the top of the list. 
Will the Gogs developers need to be involved in allowing Gogs to pass the information to the app?
